orchestrator/cli/generator/generator/product_block.py

```diff
@@ -56,16 +56,10 @@ def get_product_block_path(product_block: dict) -> Path:
 
 
 def enrich_product_block(product_block: dict) -> dict:
-    def to_block_name() -> str:
-        """Separate block name into words."""
-        type = product_block["type"]
-        name = re.sub("(.)([A-Z][a-z]+)", r"\1 \2", type)
-        return re.sub("([a-z0-9])([A-Z])", r"\1 \2", name)
-
     fields = get_all_fields(product_block)
-    block_name = product_block.get("block_name", to_block_name())
-
-    return product_block | {
+    block_name = product_block.get("block_name", product_block.get("type"))
+    return {
+        **product_block,
```

Comment:
Personally I prefer the modern (and possible faster) syntax, using the union between 2 sets with '|'

Reply:
Good to know! Fixed 👍